conn/physic/example_test.go

```diff
@@ -279,6 +373,20 @@ func ExampleEnergy_flag() {
 	flag.Parse()
 }
 
+func ExampleEnergy_float64() {
+	// BTU is used in barbecue rating. At 35000 BTU barbecue is an average power.
```

Comment:
BTU is used in barbecue rating. It is a measure of the thermal energy content of the of fuel consumed by the barbecue at its maximum rate. At 35000 BTU barbecue is an average power.